.gitignore

```diff
@@ -5,6 +5,7 @@
 /.tox
 /.coverage*
 /lbry-venv
+/venv
```

Comment:
Do we really need both `/lbry-venv` and `/venv`? I think in our docs we say to do `lbry-venv`: https://github.com/lbryio/lbry-sdk/blob/master/INSTALL.md#linuxmac

Reply:
Pycharm uses venv out of the box. Besides, it's already added in master by somebody else.

Reply:
You moved it around then, which brought it to my attention? Probably unnecessary change for this PR then?

Reply:
I guess it's fine, your cleanup is better.